pages/src/pages/notAvailable/notAvailable.tsx

```diff
@@ -0,0 +1,18 @@
+export default function NotAvailable() {
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen text-center bg-secondary">
+      <h1 className="text-9xl font-bold mb-4">Oops</h1>
```

Comment:
```suggestion
      <h1 className="text-9xl  mb-4">Oops</h1>
```
same thing about css inheritance